### PR TITLE
Add Climate, Anthology, Marvell

### PR DESCRIPTION
**Pull request by 0AYU0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -392,6 +392,9 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [r2c](https://jobs.lever.co/returntocorp/82c10097-a669-417a-9434-66a98a8bca1f) | San Francisco, CA; or Remote | Software Engineer - Internship, Application Development |
 | [Excelitas](https://www.excelitas.com/job/2022-10/software-engineering-intern) | Billerica, MA | Software Engineering Intern (No Visa Sponsorship) |
 | [Teradata](https://careers.teradata.com/jobs/215587/cloud-software-engineering-intern) | Remote | Cloud Software Engineering Intern |
+| [Climate](https://jobs.jobvite.com/careers/the-climate-corporation/job/otfWkfwI?__jvst=Job%20Board&__jvsd=LinkedIn) | San Francisco, California  Seattle, Washington  Chicago, Illinois  St. Louis, Missouri | Software Engineering Intern |
+| [Marvell](https://marvell.wd1.myworkdayjobs.com/MarvellCareers/job/US---MA---Westborough/Software-Engineering-Intern_2202008?src=SNS-102&source=LinkedIn) | Westborough, MA | Software Engineering Intern |
+| [Anthology Inc](https://careers.anthology.com/jobs/10568474-software-engineering-intern?source=LinkedIn-JS) | Remote | Software Engineering Intern |
```
Comment (ewang2002) [inline README.md]: Anthology: make it clear that "This position is not available for candidates residing in the following states: CO, CA, MA." Add something like "(Not available for residents of CO, CA, MA)"

File: README.md [hunk 2]
```diff
@@ -392,6 +392,9 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [r2c](https://jobs.lever.co/returntocorp/82c10097-a669-417a-9434-66a98a8bca1f) | San Francisco, CA; or Remote | Software Engineer - Internship, Application Development |
 | [Excelitas](https://www.excelitas.com/job/2022-10/software-engineering-intern) | Billerica, MA | Software Engineering Intern (No Visa Sponsorship) |
 | [Teradata](https://careers.teradata.com/jobs/215587/cloud-software-engineering-intern) | Remote | Cloud Software Engineering Intern |
+| [Climate](https://jobs.jobvite.com/careers/the-climate-corporation/job/otfWkfwI?__jvst=Job%20Board&__jvsd=LinkedIn) | San Francisco, California  Seattle, Washington  Chicago, Illinois  St. Louis, Missouri | Software Engineering Intern |
```
Comment (ewang2002) [inline README.md]: Separate the locations out, or just put `Multiple` instead

**Review by ewang2002 — CHANGES_REQUESTED** [review]
See suggestions. Looks good otherwise!

**Review by owini — APPROVED** [review]

**Comment by owini** [comment]
Thanks, both!

**Pull request merged**